llvm/lib/Target/AArch64/GISel/AArch64RegisterBankInfo.cpp

```diff
@@ -600,8 +600,11 @@ bool AArch64RegisterBankInfo::isLoadFromFPType(const MachineInstr &MI) const {
     EltTy = GV->getValueType();
     // Look at the first element of the struct to determine the type we are
```

Comment:
```suggestion
    // Look at the first valid element of the struct to determine the type we are
```

It seems we should also update the comment.